marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -1196,7 +1196,10 @@ const SourceToEntityMap = (props) => {
                                 placement="right"
                             ><Icon type="question-circle" className={styles.questionCircle} theme="filled" /></Popover></p>
                         </div>
-                        {emptyData ?
+                        {props.isLoading === true ? <div className={styles.spinRunning}>
+                            <MLSpin size={"large"} data-testid="spinTest"/>
```

Comment:
Can we just call spinner instead of spinTest

Reply:
flow.tsx uses spinner as data-testid and it is also used in current cypress test cases. Would it impact it if we give same testId here?